packages/client/src/widgets/Game/index.tsx

```diff
@@ -0,0 +1,226 @@
+import React, { useEffect, useRef, useState } from 'react';
+import SpaceHD from '@/widgets/Game/templates/space_hd.webp'
+
+
+interface Props {
```

Comment:
Я уже начал использовать I... для интерфейсов, T... для типов, E... для енам. На настаиваю, но предлагаю для консистентности кода
```suggestion
interface IGameProps {
```
Ну и остальным интерфейсам тоже самое